docs/nidcpower/session.rst

```diff
@@ -328,14 +328,10 @@ nidcpower.Session
    +------------------------------------------------------------+
    | :py:func:`get_channel_name`                                |
    +------------------------------------------------------------+
-   | :py:func:`get_ext_cal_last_date_and_time`                  |
```

Comment:
We shouldn't remove these, they work without cal session and they are useful in user programs.

Reply:
Missed a conditional. Added back.

Reply:
The diff still doesn't show them...

Reply:
There was yet another place where filtering was happening.

Added `filter_public_functions()` and used that for documentation.